packages/cadl-ranch-specs/http/collectionFormat/mockapi.ts

```diff
@@ -8,24 +8,33 @@ Scenarios.CollectionFormat_testMulti = passOnSuccess(
     if (req.originalRequest.originalUrl.includes("colors=blue&colors=red&colors=green")) {
       return {
         status: 200,
-        body: json({ message: `A multi collection format array was successfully received` }),
+        body: json(`A multi collection format array was successfully received`),
       };
     } else {
       return {
         status: 400,
-        body: json({ message: `Expected colors=blue&colors=red&colors=green after serialization` }),
+        body: json(`Expected colors=blue&colors=red&colors=green after serialization`),
       };
     }
   }),
 );
 
 Scenarios.CollectionFormat_testCsv = passOnSuccess(
   mockapi.get("/collectionFormat/csv", (req) => {
-    req.expect.containsQueryParam("colors", "blue,red,green");
-    return {
-      status: 200,
-      body: json({ message: `A multi collection format array was successfully received` }),
-    };
+    if (
+      req.originalRequest.originalUrl.includes("colors=blue,red,green") ||
+      req.originalRequest.originalUrl.includes("colors=blue%2Cred%2Cgreen")
```

Comment:
Just a note on this, autorest.testserver here
So I think Java encoding it since 3 years ago.
```
info: GET /queries/array/csv/string/valid?arrayQuery=ArrayQuery1%2Cbegin%21%2A%27%28%29%3B%3A%40%20%26%3D%2B%24%2C/?%23%5B%5Dend%2C%2C 200 6.420 ms - -
```